contracts/truefi2/TrueCreditAgency.sol

```diff
@@ -110,6 +105,8 @@ contract TrueCreditAgency is UpgradeableClaimable, ITrueCreditAgency {
     // mutex ensuring there's only one running loan or credit line for borrower
     IBorrowingMutex public borrowingMutex;
 
+    IPoolFactory public poolFactory;
+
```

Comment:
Do we need to add setters for these?

Reply:
We absolutely should. We'll most probably deploy the TCA without this change first

Reply:
But it could be added in a separate PR

Reply:
Unless we will deploy the different version before this one, we don't need to

Reply:
Unfortunately we will have to deploy the audited version before adding this. I'll add a setter and unit tests for this.

Reply:
If we plan to deploy audited version first and then upgrade, I believe this would break the storage

Reply:
Why would it?

Reply:
Thanks for fixing and checking this in!